library/Zend/Filter/StringToLower.php

```diff
@@ -53,7 +53,7 @@ public function filter($value)
         }
         $value = (string) $value;
 
-        if ($this->options['encoding'] !== null) {
+        if (null !== $this->getEncoding()) {
```

Comment:
There is not really a consistency when I do a search in my IDE:
- Found 403 matches of **!== null** in 205 files
- Found 265 matches of **null !==** in 129 files

There are even PR's where conditions are being changed to yoda conditions. To mention a few: #3520, #3521, #6750.